include/tvm/tir/usmp/utils.h

```diff
@@ -294,6 +319,12 @@ namespace attr {
  */
 static constexpr const char* kPoolArgs = "pool_args";
 
+/*!
+ * \brief This is a BaseFunc attribute to indicate which input var represent
```

Comment:
copy pasta from above, what is this bound to?

Reply:
Yes. good catch! fixed